renderdoc/driver/gl/gl_resources.h

```diff
@@ -29,6 +29,7 @@
 #include "gl_common.h"
 
 size_t GetCompressedByteSize(GLsizei w, GLsizei h, GLsizei d, GLenum internalformat);
+std::tuple<size_t, size_t> GetCompressedBlockSize(GLenum internalformat);
```

Comment:
Please don't use `std` data structures like this, there is `rdcpair` which is equivalent.